Installing-and-Configuring/Building.md

```diff
@@ -39,10 +60,15 @@ git clone https://github.com/osresearch/heads
 cd heads
 ```
 
-Run `make` and it will start the downloads and building process for a qemu
-emulated Heads+coreboot ROM image.  This takes a long while, so go out
-for a cup of coffee..  The initial build on a small 1-core 1GB droplet
-it will take over 90 minutes, an 8-core system takes about 40 minutes.
+Builds of Heads are reproducible, which means that Heads will build in the exact
```

Comment:
Builds of Heads **should be** reproducible, issue https://github.com/osresearch/heads/issues/734